test/integration/acceptance/get_roles_test.cpp

```diff
@@ -74,7 +74,7 @@ TEST_F(AcceptanceFixture, CanNotGetRoles) {
 
   IntegrationTestFramework(1)
       .setInitialState(kAdminKeypair)
-      .sendTx(makeUserWithPerms({}))
+      .sendTx(makeUserWithPerms({interface::permissions::Role::kGetBlocks}))
```

Comment:
Unfortunately the permission has to be removed. This requires https://github.com/hyperledger/iroha/pull/1654